hw/ip/pwrmgr/doc/_index.md

```diff
@@ -131,6 +131,27 @@ When a reset request is received during fast FSM `Active` state, the fast FSM as
 The normal power-up process described [above](#fast-clock-domain-fsm) is then followed to release the resets.
 Note in this case, the slow FSM is "not activated" and remains in its `Idle` state.
 
+### Power Manager Interal Reset Requests
```

Comment:
```suggestion
### Power Manager Internal Reset Requests
```

Reply:
fixed!

Reply:
@matutem do the contents of this PR make sense to you? Especially around `main_pok`?